.github/compatibility-test-matrices/release-v7.1.x/client-chain-b.json

```diff
@@ -0,0 +1,11 @@
+{
+  "chain-a": ["release-v7.1.x", "v6.1.0", "v5.2.0", "v4.3.0", "v4.2.0", "v4.1.1", "v3.4.0", "v3.3.1"],
+  "chain-b": ["release-v7.1.x"],
+  "entrypoint": ["TestClientTestSuite"],
+  "test": [
+    "TestClientUpdateProposal_Succeeds"
```

Comment:
we will need to add the misbehaviour test when it is merged I will try to remember this before merging!

Reply:
feel free to merge whenever you're ready and I'll add this entry to my PR!